components/Shell.tsx

```diff
@@ -268,7 +267,12 @@ function UserDropdown({ small, bottom }: { small?: boolean; bottom?: boolean })
                 "w-64 z-10 absolute mt-1 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 divide-y divide-gray-200 focus:outline-none"
               )}>
               <div className="py-1">
-                <a href={"/" + user?.username} className="flex px-4 py-2 text-sm text-neutral-500">
+                {/* note(PeerRich): please don't merge-override the hard coded cal.com */}
```

Comment:
this was merged by accident. this should not be hardcoded by using a new .env variable instead

Reply:
something like APP_URL = https://cal.com

Reply:
Done also